packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -60,6 +60,7 @@ export interface TrailProps {
   /**
    * If CloudTrail pushes logs to CloudWatch Logs in addition to S3.
    * Disabled for cost out of the box.
+   * Enabled if cloudWatchLogGroup or cloudWatchLogsRetention is provided.
```

Comment:
Thanks for adding the documentation here but we also need an update in the README.